thirdparty/EVPath/CMakeLists.txt

```diff
@@ -17,6 +17,9 @@ set(EVPATH_LIBRARY_COMPONENT adios2_evpath-libraries)
 set(EVPATH_ARCHIVE_COMPONENT adios2_evpath-development)
 set(EVPATH_HEADER_COMPONENT adios2_evpath-development)
 
+string(REGEX REPLACE "[^/]+" ".." relative_base "${EVPATH_INSTALL_MODULE_DIR}")
+list(APPEND CMAKE_INSTALL_RPATH "$ORIGIN/${relative_base}/${CMAKE_INSTALL_LIBDIR}")
+
```

Comment:
We could possibly address this in the wheel audit

Reply:
I'll look into it.

Reply:
It seems like this is going to be difficult or not possible.  I read that there is some way to tell `auditwheel` to exclude certain files, but the documentation is sparse.  By installing it in a virtual env, I can see help on the `auditwheel repair` command has:

```console
--exclude EXCLUDE     Exclude SONAME from grafting into the resulting wheel (can be specified multiple times)
```

But I don't think we want to omit it from the wheel, we just need to ignore the broken rpaths adios2 puts in those evpath libraries.  I feel like just adding a comment and/or guarding with the `ADIOS2_USE_PIP` option is all we have time for atm.

Reply:
Thanks for looking into it, it seems that is not worth it to pursue this patch

> I feel like just adding a comment and/or guarding with the ADIOS2_USE_PIP option is all we have time for atm.

Yep, that seems to be the way to go